e2e/entrypoint.sh

```diff
@@ -590,7 +590,7 @@ turn_up() {
   if [ "${EXTERNAL}" = "true" ]; then
     printf "waiting for cluster to finish coming online... "
     i=0 && while true; do
-      [ "${i}" -ge 100 ] && { error "timed out waiting for cluster" 1; return; }
+      [ "${i}" -ge 110 ] && { error "timed out waiting for cluster" 1; return; }
```

Comment:
Hi @yastij,

Hmmm, a loop iteration count of 100 equals 5 minutes. Should we just double it to 200 for 10 minutes? Any reason not to?

Reply:
I think we'll be fine with this. I can revisit if not enough, but nothing against.